release notes/v0.8.1.md

```diff
@@ -0,0 +1,20 @@
+This is a patch release that fixes a few bugs, including one [regression bug](https://github.com/grafana/xk6-browser/issues/749) introduced in v0.8.0, adds one improvement on documentation, and does minor internal modifications.
+
+
+## Bugs fixed
+
+- Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
```

Comment:
```suggestion
- Fix mapping of `Page`'s `Keyboard`, `Mouse`, and `Touchscreen` properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
```

Reply:
Fixed in: [ed6ca59](https://github.com/grafana/xk6-browser/pull/775/commits/ed6ca59561f868b2f599a83823ea6e4a2d6e742e).